dashboard/app/models/user.rb

```diff
@@ -878,7 +878,7 @@ def track_level_progress_async(script_level:, level:, new_result:, submitted:, l
       'new_result' => new_result,
       'level_source_id' => level_source_id,
       'submitted' => submitted,
-      'pairing_user_ids' => pairings ? pairings.map(&:id) : nil
```

Comment:
Note that this line causes us to not only map IDs to Users to IDs (via `pairings.map(&:id)`), but also map IDs to Users again (via `pairings`).